sdk/PowerBI.Core.Tests/Security/PowerBITokenTests.cs

```diff
@@ -126,13 +164,69 @@ public void CanCreateReportEmbedTokenWithRlsWithMultipleRoles()
         }
 
         [TestMethod]
+        public void CanCreateReportEmbedTokenWithScopes()
+        {
+            var workspaceId = Guid.NewGuid().ToString();
+            var reportId = Guid.NewGuid().ToString();
+            var scopes = Scopes.DatasetRead;
```

Comment:
If you're using the dataset read scope please change the resource to dataset id or modify the scope for the default read report one